src/backend/distributed/planner/multi_master_planner.c

```diff
@@ -30,6 +30,9 @@
 #include "utils/syscache.h"
 
 
+extern PGDLLIMPORT bool enable_hashagg;
```

Comment:
I think the right fix is to make this inside postgresql. We should open an issue to track this.

Reply:
1. We can keep this temporarily until PostgreSQL gets it. Does that work?